src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, 'Times New Roman', Times, serif;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo_link {
+  height: 40px;
+  width: 40px;
+}
+
+.nav_list {
+  height: 60px;
+  line-height: 60px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  font-size: 0;
```

Comment:
Setting `font-size: 0;` on `.nav_list` might cause the text within the list items to be invisible. Ensure this is intentional or adjust the font size to display the text correctly.